libevmasm/Instruction.h

```diff
@@ -49,6 +49,8 @@ enum class Instruction: uint8_t
 	MULMOD,				///< unsigned modular multiplication
 	EXP,				///< exponential operation
 	SIGNEXTEND,			///< extend length of signed integer
+	RETURNDATASIZE = 0xd,	///< get size of the last return data
```

Comment:
Thank you for the info.  I'll update my yellow paper PR, the test PR and this one.